pkgs/applications/editors/emacs/elisp-packages/manual-packages/copilot/package.nix

```diff
@@ -4,48 +4,37 @@
   editorconfig,
   f,
   fetchFromGitHub,
+  jsonrpc,
   replaceVars,
```

Comment:
```suggestion
```

Reply:
sorry about that, fixed.

Reply:
Do you forget to push your fix?

Reply:
I did push, but in the wrong branch :-). Sorry. Fixed.